saleor/tax/migrations/0002_add_default_tax_configs.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 3.2.13 on 2022-05-19 13:47
+
+from django.db import migrations
+
+
+def add_default_tax_class(apps, schema_editor):
```

Comment:
If we allow having `TaxClass` blank and null for `shipping` and `product` then do we need to create some basic tax class?

Reply:
We decided that we won't allow null and will always require some class, a default one or some other. If the other is removed, it will set back the default class.